### ISSUE TITLE
A fatal error has been detected by the Java Runtime Environment: Problematic frame: # V  [libjvm.so+0xbf36e2]  oopDesc::verify()+0x52

### ISSUE THREAD

**Issue by mel4tonin4** [oracle/graal#2014]
Hello.

I'm trying to switch to GraalVM for integrating Javascript in my application.

I successfully run a "Hello world!" application but I could not run my actual application to run with
```
openjdk version "11.0.5" 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-jvmci-19.3-b05-LTS)
OpenJDK 64-Bit GraalVM CE 19.3.0 (build 11.0.5+10-jvmci-19.3-b05-LTS, mixed mode, sharing)
```
while it behaves correctly under
```
openjdk version "11.0.5" 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-post-Ubuntu-0ubuntu1.1)
OpenJDK 64-Bit Server VM (build 11.0.5+10-post-Ubuntu-0ubuntu1.1, mixed mode, sharing)
```

I'm getting this error message:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f8bc3e5b6df, pid=29116, tid=29122
#
# JRE version: OpenJDK Runtime Environment (11.0.5+10) (build 11.0.5+10-jvmci-19.3-b05-LTS)
# Java VM: OpenJDK 64-Bit GraalVM CE 19.3.0 (11.0.5+10-jvmci-19.3-b05-LTS, mixed mode, sharing, tiered, jvmci, jvmci compiler, compressed oops, serial gc, linux-amd64)
# Problematic frame:
# V  [libjvm.so+0xbf36df]  oopDesc::verify()+0x4f
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P" (or dumping to /home/maurizio/development/kotlin/cli/core.29116)
#
# An error report file with more information is saved as:
# /home/maurizio/development/kotlin/cli/hs_err_pid29116.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#
./cli: line 10: 29116 Aborted                 (core dumped) java $DEBUGGING_ARGS -classpath "$(ls | xargs echo | tr " " ":")" com.aeyeonmovement.choreoplayer.cli.MainKt "$@"
```

I'm attaching the hs_err_pid29116.log file.

[hs_err_pid29116.log](https://github.com/oracle/graal/files/3991385/hs_err_pid29116.log)

Is this an issue with GraalVM? Am I doing anything wrong?

Please let me know if I can provide more information or how I can improve this report.

**Comment by dougxc** [comment]
This is a duplicate of #1915, is fixed by graalvm/labs-openjdk-11@96ba396 and will be in the GraalVM 19.3.1 release.

**Comment by mel4tonin4** [comment]
Thanks, dougxc! Season's greatings to you, the graal team and your families.
